### PR TITLE
dev/reference: fix link issues and add alter-database.md

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs-cn/pull/1451/

PTAL @lilin90

### CODE REVIEW

File: dev/reference/sql/statements/alter-database.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (lilin90) [inline dev/reference/sql/statements/alter-database.md]: I think we also need to add this file to `v3.0`?

Comment (yikeke) [reply dev/reference/sql/statements/alter-database.md]: Added, thanks!

File: v3.0/TOC.md [hunk 2]
```diff
@@ -137,6 +137,7 @@
       - [`ADD COLUMN`](reference/sql/statements/add-column.md)
       - [`ADD INDEX`](reference/sql/statements/add-index.md)
       - [`ADMIN`](reference/sql/statements/admin.md)
+      - [`ALTER DATABASE`](reference/sql/statements/alter-database.md)
```
Comment (lilin90) [inline v3.0/TOC.md]: Please also add this line to `dev`.

File: dev/reference/mysql-compatibility.md [hunk 3]
```diff
@@ -134,7 +134,7 @@ TiDB 支持 MySQL 5.7 中 **绝大多数的 SQL 模式**，以下几种模式除
 + 默认排序规则不同：
     + TiDB 中，`utf8mb4` 的默认排序规则为 `utf8mb4_bin`
     + MySQL 5.7 中，`utf8mb4` 的默认排序规则为 `utf8mb4_general_ci`，MySQL 8.0 中修改为 `utf8mb4_0900_ai_ci`
-    + 请使用 [`SHOW CHARACTER SET`](/reference/sql/statements/admin.md#-show-语句) 语句查看所有字符集的默认排序规则
+    + 请使用 [`SHOW CHARACTER SET`](/reference/sql/statements/show-character-set.md) 语句查看所有字符集的默认排序规则
```
Comment (lilin90) [inline dev/reference/mysql-compatibility.md]: Please also update it in `v3.0`.

File: dev/reference/sql/statements/alter-database.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+title: ALTER DATABASE
+summary: TiDB 数据库中 ALTER DATABASE 的使用概况。
+category: reference
+---
+
+# ALTER DATABASE
+
+`ALTER DATABASE` 用于修改指定或当前数据库的默认字符集和排序规则。`ALTER SCHEMA` 跟 `ALTER DATABASE` 操作效果一样。
+
+## Examples
```
Comment (lilin90) [inline dev/reference/sql/statements/alter-database.md]: Keep it consistent with other similar files.

```suggestion
## 示例
```

File: dev/reference/sql/statements/alter-database.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+ALTER {DATABASE | SCHEMA} [db_name]
+    alter_specification ...
+alter_specification:
+    [DEFAULT] CHARACTER SET [=] charset_name
+  | [DEFAULT] COLLATE [=] collation_name
+```
+
+`alter_specification` 选项用于指定数据库具体的 `CHARACTER SET` 和 `COLLATE`。目前 TiDB 只支持部分的字符集和排序规则，请参照[字符集支持](/reference/sql/character-set.md)。
+
+## See also
```
Comment (lilin90) [inline dev/reference/sql/statements/alter-database.md]:
```suggestion
## 另请参阅
```

File: v3.0/reference/sql/statements/alter-database.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+title: ALTER DATABASE
+summary: TiDB 数据库中 ALTER DATABASE 的使用概况。
+category: reference
+---
+
+# ALTER DATABASE
+
+`ALTER DATABASE` 用于修改指定或当前数据库的默认字符集和排序规则。`ALTER SCHEMA` 跟 `ALTER DATABASE` 操作效果一样。
+
+## Examples
```
Comment (lilin90) [inline v3.0/reference/sql/statements/alter-database.md]:
```suggestion
## 示例
```

File: v3.0/reference/sql/statements/alter-database.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+ALTER {DATABASE | SCHEMA} [db_name]
+    alter_specification ...
+alter_specification:
+    [DEFAULT] CHARACTER SET [=] charset_name
+  | [DEFAULT] COLLATE [=] collation_name
+```
+
+`alter_specification` 选项用于指定数据库具体的 `CHARACTER SET` 和 `COLLATE`。目前 TiDB 只支持部分的字符集和排序规则，请参照[字符集支持](/reference/sql/character-set.md)。
+
+## See also
```
Comment (lilin90) [inline v3.0/reference/sql/statements/alter-database.md]:
```suggestion
## 另请参阅
```

**Comment by yikeke** [comment]
PTAL again~ @lilin90

**Pull request merged**